### PR TITLE
Chore/panel colour

### PR DESCRIPTION
**Pull request by RichardHarperInfinityWorks**
Update panel background colour to match GDS

GDS:

Screenshot 2023-09-29 at 10 56 04

Link to the Gov Panel Component https://design-system.service.gov.uk/patterns/confirmation-pages/

**Checklist**:

* [ ] Documentation
* [ ] Tests
* [ ] Ready to be merged

### CODE REVIEW

**Comment by RichardHarperInfinityWorks** [comment]
@penx could you please review this update on the panel colour to match the updated Style guide, thanks

File: components/panel/src/index.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const StyledPanel = styled('div')(
     },
 
     color: WHITE,
-    background: TURQUOISE,
+    background: BUTTON_COLOUR,
```
Comment (penx) [inline components/panel/src/index.tsx:34]: I would prefer we either hard code this to '#00703c' here or do an update to govuk-colours to export a semantic name that matches whatever is in govuk-frontend.

Probably the former as I have limited time and no support to maintain this at the moment

**Comment by penx** [comment]
Thanks for the PR! 1 comment, if we could hard code this then I think it can go in.

Please can you link to an official govuk design system page showing this colour being used on a panel in the PR description?

File: components/panel/src/index.tsx (hunk 1 above)
Comment (RichardHarperInfinityWorks) [reply components/panel/src/index.tsx]: @penx updated to hex as requested, thanks for the prompt response

**Comment by RichardHarperInfinityWorks** [comment]
> Thanks for the PR! 1 comment, if we could hard code this then I think it can go in.
> 
> Please can you link to an official govuk design system page showing this colour being used on a panel in the PR description?

Added the link as requested

**Comment by RichardHarperInfinityWorks** [comment]
> Thanks for the PR! 1 comment, if we could hard code this then I think it can go in.
> 
> Please can you link to an official govuk design system page showing this colour being used on a panel in the PR description?

Add the Link as requested

**Review by penx — APPROVED** [review]

**Comment by RichardHarperInfinityWorks** [comment]
@penx  I have updated the test to ensure the panel colour matches as it is currently failing

**Comment by RichardHarperInfinityWorks** [comment]
@penx looks like it's ready to merge

**Comment by RichardHarperInfinityWorks** [comment]
@penx what is the release process to get a 10.6 built and available?

**Comment by penx** [comment]
The process is me finding some time to do it on my evenings or weekends, despite the only support I get for this project being (much appreciated) community PRs such as this 😀

I'll see if I can find some time this weekend or one evening next week to do a release.

If you, your consultancy, or a government department you work for are using this in production could I request that someone considers sponsoring this project?

I'll continue to support where I can regardless, but there won't always be fast turnarounds on PR responses or releases unless this changes, sorry.

**Comment by RichardHarperInfinityWorks** [comment]
> The process is me finding some time to do it on my evenings or weekends, despite the only support I get for this project being (much appreciated) community PRs such as this 😀
> 
> I'll see if I can find some time this weekend or one evening next week to do a release.
> 
> If you, your consultancy, or a government department you work for are using this in production could I request that someone considers sponsoring this project?
> 
> I'll continue to support where I can regardless, but there won't always be fast turnarounds on PR responses or releases unless this changes, sorry.

Have sent you some $$$$$, thanks for maintaining this.

**Comment by penx** [comment]
@RichardHarperInfinityWorks thank you! 

I released 0.10.6 yesterday. Let me know if you have any issues.

**Pull request merged**